### DISCUSSION TITLE
LinkAdder::extractLinksFromHtml()

### DISCUSSION THREAD

**Discussion by Xaelp** [spatie/crawler#455] [General]
I just found out that the crawler misses links provided through **HTML form element action** attribute. 

Would it make sense to add a change that supports configuring it so that it is correctly handled through the **/src/LinkAdder::extractLinksFromHtml()** method?

**Comment by cth-latest** [comment]
Hello @Xaelp,

I just submitted a PR about this https://github.com/spatie/crawler/pull/458. This should be possible soon !
